frontend/src/store/modules/storage.js

```diff
@@ -7,7 +7,9 @@
 import Vue from 'vue'
 
 // initial state
-const state = {}
+const state = {
+  'global/color-scheme': 'auto'
+}
```

Comment:
```suggestion
const state = {}
```
The state of the storage module is initialized in the storage plugin